api/cf/services.go

```diff
@@ -0,0 +1,65 @@
+package cf
```

Comment:
Just learned that https://github.com/cloudfoundry-community/go-cfenv is a thing. Totally fine with this for now, but consider swapping this out in favor of `go-cfenv` in the future.